repositories/src/test/scala/cromwell/pipeline/datastorage/dao/utils/PostgreTestSources.scala

```diff
@@ -0,0 +1,37 @@
+package cromwell.pipeline.datastorage.dao.utils
+
+import com.dimafeng.testcontainers.{ ForAllTestContainer, PostgreSQLContainer }
+import com.typesafe.config.Config
+import cromwell.pipeline.datastorage.DatastorageModule
+import cromwell.pipeline.utils.{ ApplicationConfig, TestContainersUtils }
+import org.scalatest.{ AsyncWordSpec, BeforeAndAfterAll, BeforeAndAfterEach }
+import slick.jdbc.PostgresProfile.api._
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+trait PostgreTestSources extends AsyncWordSpec with ForAllTestContainer with BeforeAndAfterEach with BeforeAndAfterAll {
+
+  override val container: PostgreSQLContainer = TestContainersUtils.getPostgreSQLContainer()
+  private lazy val config: Config = TestContainersUtils.getConfigForPgContainer(container)
+  lazy val datastorageModule: DatastorageModule = new DatastorageModule(ApplicationConfig.load(config))
+
+  import datastorageModule.pipelineDatabaseEngine
+
+  override protected def beforeAll(): Unit = {
+    super.beforeAll()
+    pipelineDatabaseEngine.updateSchema()
+  }
+
+  override protected def afterEach(): Unit = {
+    super.afterEach()
+    cleanDatabase()
+  }
+
+  private def cleanDatabase(): Unit =
+    Await.result(
+      pipelineDatabaseEngine.database.run(sqlu"""TRUNCATE TABLE "user", project, run RESTART IDENTITY;"""),
+      Duration.Inf
```

Comment:
Are we can change Infinity to a real time, like `Duration.ofSeconds(5)` or something else ?
You shold think about it.

Reply:
I definitely would! Thanks. So what is a more appropriate duration. Maybe also some "not responding" cases need additional handling?

Reply:
@Nikolai-Isaev take a look at `cromwell.pipeline.utils.TestTimeout`

Reply:
Wow we have the whole trait for that, my bad )

Reply:
Fixed